.idea/GroceryBuddy.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Probably want to set .gitignore to ignore all the files in .idea/

Reply:
Good suggestion we'll add that in. Hopefully, this doesn't happen in the future as we are using vscode and not IDEA as the ide for this project.